socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -61,31 +58,59 @@ export default Vue.component("author-bar", {
         isUserAuthentificated: {type: Boolean, required: true},
     },
     data() {
-        return {showProfileBox: false}
+        return {
+            showProfileBox: false,
+            _isUserFollowingAuthor: this.isUserFollowingAuthor,
+        }
     },
     computed: {
+        isUserRemote(){
+            return !this.isUserLocal
+        },
         showFollowBtn() {
-            return this.isUserAuthentificated
-                    && !this.isUserLocal
-                    && !this.isUserFollowingAuthor
+            return this.canFollow && !this.$data._isUserFollowingAuthor
```

Comment:
> Is `this.$data` the correct way to do this?

Yes. As told, earlier, the `_` prefixed data items work differently. A bit like the dunder methods in Python. They're not directly accessible. This is a convention to mark internal variable but it is enforced by the framework itself. It seems not documented, but this explenation is available [here](https://forum.vuejs.org/t/underscore-prefixes-for-vue-data/4018).

Though, I'm not confortable with this piec of code. It's basically a hack because properties must not be mutated. Props mutations get erased after each parent update. So passing this information via props seems not the way to go. A store, possibly? Or [`provide/inject`](https://vuejs.org/v2/api/#provide-inject)?